lib/resources/app_theme.dart

```diff
@@ -23,4 +25,11 @@ const MaterialColor colorWhite = const MaterialColor(
     800: const Color(0xFFFFFFFF),
     900: const Color(0xFFFFFFFF),
   },
-);
\ No newline at end of file
+);
+
+void setWhiteStatusBar() {
```

Comment:
let's make it a bit more generic by passing the color as an argument.
this will make it more understandable and we won't need to duplicate code in routes with specific status bar color like in `profile_route`
`SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
        statusBarColor: AppColors.colorBlackGray,
        statusBarIconBrightness: Brightness.light));`